backend/YouTubeV2.Application/Services/ISubscriptionService.cs

```diff
@@ -6,5 +6,7 @@ public interface ISubscriptionService
     {
```

Comment:
The whole authorization still doesnt work! This is super imortant so for now ignore the subscriptions who cares about them rn and fix the authentication. Im getting 401 after login on frontend

Reply:
It works for me in swagger. You need to put Bearer *key*. 
![image](https://user-images.githubusercontent.com/45049508/232623603-efcc78af-3f83-488c-b49a-1e958b24871a.png)